web/frontend/src/components/movement-sensor/index.svelte

```diff
@@ -106,6 +116,35 @@ useConnect(() => {
               {altitudeM?.toFixed(2)}
             </td>
           </tr>
+          {#if accuracy?.positionNmeaGgaFix}
+          <tr>
+            <th class="border border-medium p-2"> NMEA Fix Quality </th>
+            <td class="border border-medium p-2">
+              {accuracy.positionNmeaGgaFix}
+              {#if accuracy.positionNmeaGgaFix === 1 || accuracy.positionNmeaGgaFix === 2}
+                <div>expect 1m-5m accuracy</div>
+              {/if}
+              {#if accuracy.positionNmeaGgaFix === 4 || accuracy.positionNmeaGgaFix === 5}
+                <div>expect 2cm-50cm accuracy</div>
+              {/if}
+            </td>
+          </tr>
+          {/if}
+
+          {#if accuracy?.positionHdop && accuracy?.positionVdop}
```

Comment:
Why are accuracy values being added to the position table?

Reply:
positionHDOP, positionVDOP and NmeaGggaFix are positional accuracies that are associated with GPS position. It made sense to keep the accuracy details with the position.

Reply:
I feel like it is sufficient to have them in the accuracy table, but I also don't have a full understanding of how these values would be used in practice by our users. What do you expect users to do with this info?

Reply:
The idea behind it was so the user can use the HDOP/VDOP values to understand how accurate longitude and latitude is. Motion planning has been using this value to determine the accuracy of the GPS coordinates along with the quality of the positional data. I expect other users who will be using GPS would use these accuracy values similarly. And since it's always related to the position, it made sense to keep them together under position.

Reply:
Sounds good to me!